### ISSUE TITLE
Skype context menus not read

### ISSUE THREAD

**Issue by nvdaTrac** [nvaccess/test_nvdaTracImport#250]
**Reported by jrutkowski on 11 Sep 2008 12:28 UTC**
When pressing the applications key in Skype, NVDA fails to read the options in the context menus associated with contacts.

**Comment by nvdaTrac** [comment]
**Comment by jteh on 11 Sep 2008 12:36 UTC**
Do you mean the context menu for the individual contacts in the contact list? This reads fine for me. Please ensure that you have updated to the latest version of Skype and that View -> Accessibility -> Extended keyboard navigation is enabled.

**Comment by nvdaTrac** [comment]
**Comment by Bernd on 29 Oct 2008 15:59 UTC**
Replying to [jteh](comment:1):
> Do you mean the context menu for the individual contacts in the contact list? This reads fine for me. Please ensure that you have updated to the latest version of Skype and that View -> Accessibility -> Extended keyboard navigation is enabled.

I thik there is a FAQ-article in the Wiki so this ticket could be closed as invalid. If not please reopen

**Comment by nvdaTrac** [comment]
**Comment by Agent Golder on 29 Dec 2009 15:07 UTC**
Anyway, no other screen reader does this except NVDA.
Can't it be fixed? Is it so difficult for you the programmers?
I don't have any related knolage but I think this is a mager defect.
